src/index.ts

```diff
@@ -31,7 +35,8 @@ const binaryMimeTypes = [
 const server = awsServerlessExpress.createServer(expressServer.app, undefined, binaryMimeTypes);
 
 exports.handler = (event: AWSLambda.APIGatewayEvent, context: AWSLambda.Context) => {
-  awsServerlessExpress.proxy(server, event, context);
-  console.log(JSON.stringify(event));
-  console.log('hello');
+  logger.debug(JSON.stringify(event));
+  logger.debug('hello');
+
+  return awsServerlessExpress.proxy(server, event, context);
```

Comment:
bug fix: result of awsServerlessExpress.proxy() should be returned in aws lambda.